src/core/ddsi/src/q_plist.c

```diff
@@ -951,6 +1027,15 @@ static dds_return_t dvx_reader_favours_ssm (void * __restrict dst, const struct
 }
 #endif
 
+/* pserop representation of dds_propertyseq_t. */
```

Comment:
I'm not so keen on doing these with macros, given that they are all used exactly once.